Code/skyrim_ui/src/app/components/chat/chat.component.html

```diff
@@ -2,13 +2,13 @@
   <p
     #entry
     *ngFor="let message of messages"
-    [attr.data-system]="!message.name"
     [attr.data-odd]="message.odd"
```

Comment:
could you replace the odd attribute of the message with the odd alias of the ngFor

Reply:
Even better